transforms/src/button-v0-to-v1.ts

```diff
@@ -0,0 +1,110 @@
+import { Transform, JSCodeshift, FileInfo, API, Options, ImportDeclaration, ASTPath, JSXElement } from 'jscodeshift';
+
+export const transform: Transform = (fileInfo: FileInfo, api: API, options: Options) => {
+  const j: JSCodeshift = api.jscodeshift;
```

Comment:
Is there a precedent for this one letter variable? Otherwise, I'm not quite a fan of it..

Reply:
Yeah, it's pretty common for jscodeshift code: 

https://github.com/facebook/jscodeshift#parser
https://github.com/skovy/font-awesome-codemod/blob/master/transforms/implicit-icons-to-explicit-imports.ts#L140